integration-testing/src/test/kotlin/kotlinx.atomicfu.gradle.plugin.test/framework/runner/Environment.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2016-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.atomicfu.gradle.plugin.test.framework.runner
+
+import java.io.File
+
+internal const val ENABLE_JVM_IR_TRANSFORMATION = "kotlinx.atomicfu.enableJvmIrTransformation"
+internal const val ENABLE_JS_IR_TRANSFORMATION = "kotlinx.atomicfu.enableJsIrTransformation"
+internal const val ENABLE_NATIVE_IR_TRANSFORMATION = "kotlinx.atomicfu.enableNativeIrTransformation"
+
+// todo: get from System.properties
+internal val atomicfuVersion = "0.22.0-SNAPSHOT"
```

Comment:
Can we somehow get `atomicfuVersion` from `integration-testing/gradle.properties`, maybe as a System property